Sources/Commands/PackageCommands/AddDependency.swift

```diff
@@ -50,33 +51,49 @@ extension SwiftPackageCommand {
         @Option(help: "Specify upper bound on the package version range (exclusive)")
         var to: Version?
 
+        @Option(help: "Specify dependency type (url, path, registry)")
+        var type: DependencyType = .url
+
+        enum DependencyType: String, Codable, ExpressibleByArgument {
```

Comment:
Add CaseIterable so the possible values are displayed by ArgumentParser.

Reply:
fixed